### PR TITLE
benchmark

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

**Comment by ashtul** [comment]
@gkorland @itamarhaber @K-Jo 
If someone can run this today to confirm it is OK and merge, it will be great.
I link to it in the blog post.

**Comment by K-Jo** [comment]
@swilly22 can you please review this?

File: tests/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+for line in page.iter_lines():
+    for word in line.split():
+        redis_pipe.zincrby('bm_text', 1, word)
+
+responses = redis_pipe.execute()
+for response in responses:
+	pass
+
+real_results = redis.zrevrange('bm_text', 0, 49)
+print('Memory used %s'% redis.memory_usage('bm_text'))
```
Comment (swilly22) [inline tests/benchmark.py]: Stop timer here.

File: tests/benchmark.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+print('This is an accurate list for comparison')
+print("--- %s seconds ---" % (time.time() - start_time))
+print(redis.zcount('bm_text', '-inf', '+inf'))
+start_time = time.time()
+
+# test Top-K
+print("K Width(*k) Depth Memory Accuracy Time")
+k_list = [10, 50, 100, 1000]
+pipe = redis.pipeline()
```
Comment (swilly22) [inline tests/benchmark.py]: Reuse existing pipeline

File: tests/benchmark.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+responses = redis_pipe.execute()
+for response in responses:
+	pass
+
+real_results = redis.zrevrange('bm_text', 0, 49)
+print('Memory used %s'% redis.memory_usage('bm_text'))
+print('This is an accurate list for comparison')
+print("--- %s seconds ---" % (time.time() - start_time))
+print(redis.zcount('bm_text', '-inf', '+inf'))
+start_time = time.time()
```
Comment (swilly22) [inline tests/benchmark.py]: Seems unnecessary, see line 51.

File: tests/benchmark.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+for k in k_list:
+	real_results = redis.zrevrange('bm_text', 0, k - 1)
+	for width in [4, 8]:
+		for depth in [3, 7, 10]:
+			redis.execute_command('DEL', 'bm_topk')
+			start_time = time.time()
+			create_topk(redis, k, width, depth)
+
+			for line in page.iter_lines():
+				if line is not '' and line is not ' ':
```
Comment (swilly22) [inline tests/benchmark.py]: Why isn't this check performed after line 27?

Comment (ashtul) [reply tests/benchmark.py]: B/c it didn't scream at me :)
Added it for fairness of benchmark

**Comment by ashtul** [comment]
@swilly22 @K-Jo 
Good comments. Fixed!

**Pull request merged**